spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/hive/HivePropertiesConverter.java

```diff
@@ -6,19 +6,121 @@
 package com.datastrato.gravitino.spark.connector.hive;
 
 import com.datastrato.gravitino.spark.connector.PropertiesConverter;
-import java.util.HashMap;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
+import java.util.Locale;
 import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.ws.rs.NotSupportedException;
+import org.apache.spark.sql.connector.catalog.TableCatalog;
 
-/** Transform hive catalog properties between Spark and Gravitino. Will implement in another PR. */
+/** Transform hive catalog properties between Spark and Gravitino. */
 public class HivePropertiesConverter implements PropertiesConverter {
 
+  // Transform Spark hive file format to Gravitino hive file format
+  static final Map<String, String> fileFormatMap =
+      ImmutableMap.of(
+          "sequencefile", HivePropertiesConstants.GRAVITINO_HIVE_FORMAT_SEQUENCEFILE,
+          "rcfile", HivePropertiesConstants.GRAVITINO_HIVE_FORMAT_RCFILE,
+          "orc", HivePropertiesConstants.GRAVITINO_HIVE_FORMAT_ORC,
+          "parquet", HivePropertiesConstants.GRAVITINO_HIVE_FORMAT_PARQUET,
+          "textfile", HivePropertiesConstants.GRAVITINO_HIVE_FORMAT_TEXTFILE,
+          "json", HivePropertiesConstants.GRAVITINO_HIVE_FORMAT_JSON,
+          "csv", HivePropertiesConstants.GRAVITINO_HIVE_FORMAT_CSV,
+          "avro", HivePropertiesConstants.GRAVITINO_HIVE_FORMAT_AVRO);
+
+  static final Map<String, String> sparkToGravitinoPropertyMap =
+      ImmutableMap.of(
+          "hive.output-format",
+          HivePropertiesConstants.GRAVITINO_HIVE_OUTPUT_FORMAT,
+          "hive.input-format",
+          HivePropertiesConstants.GRAVITINO_HIVE_INPUT_FORMAT,
+          "hive.serde",
+          HivePropertiesConstants.GRAVITINO_HIVE_SERDE_LIB);
+
+  /**
+   * CREATE TABLE xxx STORED AS PARQUET will save "hive.stored.as" = "PARQUET" in property.
```

Comment:
Is it "hive.stored-as" or "hive.stored.as"?

Reply:
`hive.stored-as`, will correct it